app/controllers/provider/sessions_controller.rb

```diff
@@ -101,4 +102,8 @@ def session_return_to
   def instantiate_sessions_presenter
     @presenter = Provider::SessionsPresenter.new(domain_account)
   end
+
+  def redirect_to_sso_params
```

Comment:
This name is neither correct nor understandable IMO...
Maybe we can go for something like `redirect_to_enforced_sso`